src/Microsoft.Health.Fhir.Core/Features/Security/DataActions.cs

```diff
@@ -23,8 +23,9 @@ public enum DataActions : ulong
         ResourceValidate = 1 << 5,
         Reindex = 1 << 6,
         ConvertData = 1 << 7,
+        ProfileAdmin = 1 << 8, // Allows to Create/Update/Delete resources related to profile's resources.
 
         [EnumMember(Value = "*")]
-        All = (ConvertData << 1) - 1,
+        All = (ProfileAdmin << 1) - 1,
```

Comment:
Happy to hear better name suggestions :)

Reply:
What about `ProfileEditor` ?

---
In reply to: [576425352](https://github.com/microsoft/fhir-server/pull/1666#discussion_r576425352) [](ancestors = 576425352)

Reply:
Sounds quite the same to me. I'm mostly unhappy with word `Profile` since it's not exactly profile, it's just bunch of resources used during profile validation

Reply:
I feel that `Admin` usually refers to the person/role who is in charge of the whole server. Hence the preference for `Editor`. Also, is it expected that a user who does CUD operations on profile also follows it up with a reindex operation? If so, would it make sense to combine the roles into a single one instead of multiple roles?

---
In reply to: [577313196](https://github.com/microsoft/fhir-server/pull/1666#discussion_r577313196) [](ancestors = 577313196)

Reply:
How long do we want these names to be? Something like `ProfileDefinitionsEditor` adds more explanation.